index.js

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env node
+
+const axios = require('axios');
+const path = require('path');
+const fs = require('fs-extra');
+const program = require('commander');
+
+let input;
+
+program
+    .name('icaredata-client')
+    .usage('<path-to-messages> [options]')
+    .option('-o, --out <out>', 'the path to the output folder', path.join('.', 'out'))
+    .arguments('<path-to-messages>')
+    .action(function (pathToMessages) {
+        input = pathToMessages;
+    })
+    .parse(process.argv);
+
+// Check that input folder is specified
+if (!input) {
+    console.error('Missing path to messages folder.');
+    program.help();
+}
+
+// Try to parse the files from the input directory
+let files;
+try {
+    files = fs.readdirSync(input, 'utf8');
+} catch (e) {
+    console.error('Invalid path to messages folder.');
+    program.help();
+}
+
+// Collect the message files and the config file
+const data = { config: null, messages: [] };
+for (const file of files) {
+    const fileContent = fs.readFileSync(path.join(input, file), 'utf8');
```

Comment:
It might also be nice to validate that we're not attempting to read a directory here too

Reply:
Good call, on it!